src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Streams/Given_RandomAccessStreamReference.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_Storage.Streams
+{
+	[TestClass]
+	public class Given_RandomAccessStreamReference
+	{
+		private const string _unoStaticTestFileContent = "https://platform.uno/\r\n";
+
+		[TestMethod]
+		public async Task When_FromUri()
+		{
+			var sut = RandomAccessStreamReference.CreateFromUri(new Uri("https://nv-assets.azurewebsites.net/uno-unit-tests.txt"));
+			var actual = await ReadToEnd(sut);
+
+			Assert.AreEqual(_unoStaticTestFileContent, actual);
+		}
+
+#if !NETFX_CORE
+		[TestMethod]
+		public async Task When_FromFile()
+		{
+			var temp = await GetTempFile();
+			var sut = RandomAccessStreamReference.CreateFromFile(temp); // We create the ref even before writing anything in it.
+
+			var tempContent = Guid.NewGuid().ToString("N");
+			var tempOutStream = await temp.OpenStream(CancellationToken.None, FileAccessMode.ReadWrite, StorageOpenOptions.AllowReadersAndWriters);
+			using (var writer = new StreamWriter(tempOutStream))
+			{
+				writer.Write(tempContent);
+			}
+
+			var actual = await ReadToEnd(sut);
+
+			Assert.AreEqual(tempContent, actual);
+		}
+#endif
+
+		[TestMethod]
+		public async Task When_FromStream()
+		{
+			var temp = new MemoryStream();
+			var sut = RandomAccessStreamReference.CreateFromStream(temp.AsRandomAccessStream()); // We create the ref even before writing anything in it.
+
+			var tempContent = Guid.NewGuid().ToString("N");
```

Comment:
Codacy found an issue: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6307767)